### DISCUSSION TITLE
Material do Array - 13

### DISCUSSION THREAD

**Discussion by DouglasSoares1203** [balta-io/2801#81] [Perguntas e Respostas] [answered]
Bom dia

Verifiquei que o material do Array não consta para baixar os slides, vcs poderiam passar o material fisico para baixar?

Att,

Douglas.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @DouglasSoares1203, como vai? Nem todas as aulas tem o PPT presente no repositório... Infelizmente não temos como disponibilizar este PPT no momento.
